content/posts/2024-02-06-der-sprudelfaktor-monatsbericht-januar-2024.de.md

```diff
@@ -0,0 +1,60 @@
+---
+sticky: false
+title: Der Sprudelfaktor – Monatsbericht Januar 2024
+author: Community
+date: 2024-02-06T21:55:23.000Z
+draft: false
+categories:
+  - news
+  - monatsbericht
+  - dsee
+tags:
+  - umbau
+  - transparenz
+featured_image: /wp-content/uploads/2024/02/monatsbericht_naehcafe.jpg
+---
+
+Schon wieder ist ein Monat vorbei, Zeit für den Rückblick auf den Januar! Mit **18 öffentlichen Veranstaltungen**, einem offiziellen Besuch bei uns und mindestens einem Auswärtstermin haben wir zum Jahresauftakt gleich einen neuen Rekord geschafft.
+
+Ganz regelmäßig finden die zweiwöchentlichen **Maker Mondays** statt, und bei beiden Terminen waren sie wieder ein Publikumsmagnet mit je 30 bis 40 Anwesenden, die in der Holzwerkstatt, an den 3D-Druckern oder am Lasercutter arbeiteten. Am 8.1. fiel der Makermonday zudem mit dem **Nähcafé** zusammen, so dass vorne geschnitten und genäht wurde, während hinten im Haus Fräsen und Leimen dran waren. Drei Mal traf sich die Gruppe um **The Things Network/LoRaWAN** zu ihren offenen Abenden, zwei Mal ging es bei **Capture The Flag/PPP** um die spielerische Annäherung an Prinzipien der IT-Sicherheit, und fünf Mal beschäftigten wir uns bei **Live Let's Plays** mit aktueller Computerspielekultur.
```

Comment:
```suggestion
Ganz regelmäßig finden die zweiwöchentlichen **Maker Mondays** statt, und bei beiden Terminen waren sie wieder ein Publikumsmagnet mit je 30 bis 40 Anwesenden, die in der Holzwerkstatt, an den 3D-Druckern oder am Lasercutter arbeiteten. Am 8.1. fiel der Makermonday zudem mit dem **Nähcafé** zusammen, so dass vorne geschnitten und genäht wurde, während hinten im Haus Fräsen und Leimen dran waren. Drei Mal traf sich die Gruppe um **The Things Network/LoRaWAN** zu ihren offenen Abenden, zwei Mal ging es bei **Capture The Flag/PPP** um die spielerische Annäherung an Prinzipien der IT-Sicherheit, und fünf Mal beschäftigten wir uns bei [**Live Let's Plays**](https://defcon.social/@LLP222) unter anderem damit, welche [grundlegenden Bedürfnisse](https://defcon.social/@LLP222/111716672458159793) ein Game erfüllen muss, und wie das Verhältnis m:w bei Gamer:innen aussieht.
```

Reply:
Hm, magst du für den Angebotslink nicht eher mal eine Gruppe auf https://temporaerhaus.de/gruppen/ anlegen, @gitbreaker222?

Reply:
hm da ich das schon an einigen Orten gemacht habe (angefangen in Stuttgart) und auch alleine organisiere, sehe ich das noch als ein persönliches Ding. 
Kann mir aber schon vorstellen, das in eine Gruppe zu integrieren, sobalt wirklich mehr Leute mit games was machen. Konsti hatte schon mal ne LAN angedacht und ich hab in der Pipeline noch ein Cozy-Gaming-Sonntags-Format, das ich gerne anstoßen und direkt weiter geben möchte... 🔜 Soon

Reply:
Zu den Gruppen: Machen wir das nicht alle irgendwie als persönliches Ding? 😜 
Zu deinem Vorschlag: Ich bin kein Freund von uneingeführten Abkürzungen, die führen gern zu Missverständnissen. Können wir das eventuell ausschreiben?

Reply:
welche Abkürzung meinst du? Mach gern nen PR mit deinem Vorschlag auf, dann können wir es ergänzen :)

Reply:
```suggestion
Ganz regelmäßig finden die zweiwöchentlichen **Maker Mondays** statt, und bei beiden Terminen waren sie wieder ein Publikumsmagnet mit je 30 bis 40 Anwesenden, die in der Holzwerkstatt, an den 3D-Druckern oder am Lasercutter arbeiteten. Am 8.1. fiel der Makermonday zudem mit dem **Nähcafé** zusammen, so dass vorne geschnitten und genäht wurde, während hinten im Haus Fräsen und Leimen dran waren. Drei Mal traf sich die Gruppe um **The Things Network/LoRaWAN** zu ihren offenen Abenden, zwei Mal ging es bei **Capture The Flag/PPP** um die spielerische Annäherung an Prinzipien der IT-Sicherheit, und fünf Mal beschäftigten wir uns bei [**Live Let's Plays**](https://defcon.social/@LLP222) mit verschiedenen Themen rund um aktuelle Computerspielekultur.
```

Reply:
Habe das jetzt knapp 10 Minuten lang angestarrt und bin für mich zu dem Schluss gekommen, dass ich ungern allen anderen Veranstaltungen in dem Absatz jeweils nur 2-4 Worte widmen würde während LLP 2 Links und ausführliche Beschreibung zweier Detailthemen bekommt. Liest sich außerdem bisschen holprig; Gamer:innen verwendet : anstatt *, m:w Verhältnis is wiederum spezifisch auf die beiden Geschlechter limitiert, und es ist nicht ganz klar was es heißen soll dass ein Spiele "Bedürfnisse erfüllen muss" (für was? Ohne Link ergibt das wenig Sinn, und wenn man's noch in den Text mit rein macht, wird es noch länger als es ohnehin schon im Vergleich zu den anderen Veranstaltungen ist).

Insofern hier mein Vorschlag. Ähnlich dem ursprünglichen Text, mit einem Link und 2 Worte länger.

Reply:
Genau das mit "m:w" war die Abkürzung, die mich gestört hat. Deine Fassung @ghabiger find ich gut - ich würds jetzt erstmal so raushauen und dann können wir immer noch schauen, ob wir nachjustieren.

Reply:
ah ein paar Erläuterungen:
- gendergap mit `:` habe ich mir persönlich so angewöhnt. Kann das gerne zu `*` ändern, wenns besser zum tph-Stil passt
- m:w basiert auf der Statistik [aus dem Buch](https://defcon.social/@LLP222/111815982501282803), das ich bei einem Intro mal vorgestellt habe.

Bei meinem Vorschlag wollte ich hervorheben, dass wir nicht _nur_ spielen, sondern uns jede Woche mit konkreten Themen beschäftigen. Damit das nicht nur eine Behauptung ist, habe ich zwei von vier Beispielen ohne besondere Gewichtung raus gepickt. Wie könnte man das besser einbringen?